app/app.css

```diff
@@ -23,3 +23,23 @@
   content: "|";
   padding-right: 0.3em;
 }
+
+
+
+.table > thead > tr> th{text-align: center;}
+.table > tbody > tr> td{text-align: center;}
+.btn.btn-default.btn-block{text-align: center;}
```

Comment:
I'm assuming that you are trying to change some bootstrap, this is not the way to do that. I'd have to take a look at what your dong in order to give a way to fix it.

Reply:
The tables had text that were not centered so I just found the element where it can be changed and centered it